src/main/java/org/mozilla/zest/impl/HttpClientWrapper.java

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.zest.impl;
+
+import org.apache.commons.httpclient.*;
```

Comment:
Use specific imports, easier to check which classes are in use, also, star imports are not used throughout the codebase.